paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -294,15 +302,18 @@ def notify_after_auto_rollback(self):
         if self.old_commit is not None and self.commit != self.old_commit:
             message = (
                 f"*{self.service}* - Deployment of {self.commit} for {self.deploy_group} *failed*!\n"
-                f"Auto-rolling back to {self.old_commit}"
+                f"Auto-rolling back to {self.old_commit}\n"
+                f"{self.url_message}\n"
             )
             self.post(channels=self.channels, message=message)
 
     def notify_after_abort(self):
         if self.old_commit is not None and self.commit != self.old_commit:
             message = (
                 f"*{self.service}* - Deployment of {self.commit} to {self.deploy_group} *aborted*.\n"
-                "PaaSTA will keep trying to deploy this code until it is healthy."
+                "PaaSTA will keep trying to deploy this code until it is healthy.\n"
```

Comment:
suggestion: "Commit {self.commit} is still marked for deployment; PaasTA will keep trying to deploy it until it is healthy.  To roll back, use paasta rollback"